app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
+    session[:question_number] = 0
+    render 'question', locals: {quest: session[:resolution].questions[session[:question_number]]}
   end
```

Comment:
It seems like the proofer gem is expecting the client to keep track of questions and store various things in the session. What about reworking the gem to return some sort of response with a status to let the client know how to proceed, similarly to how IdPaaS works?